main.tf

```diff
@@ -67,6 +67,14 @@ resource "azurerm_kubernetes_cluster" "main" {
     }
   }
 
+  role_based_access_control {
+    enabled = var.enable_role_based_access_control
+    azure_active_directory {
+      managed                = true
+      admin_group_object_ids = var.aad_admin_group_object_ids
```

Comment:
Would you add another option when `managed` is `false`, and `client_app_id`, `server_app_id` and `server_app_secret` is set?

Reply:
Done